func_sig_registry/registry/models.py

```diff
@@ -90,6 +90,7 @@ def import_from_raw_text_signature(cls, raw_function_signature):
             text_signature = normalize_function_signature(raw_function_signature)
         except ValueError:
             logger.error("error signature: %s", raw_function_signature)
+            return None, False
```

Comment:
Just repeating what I stated in the other issue.  Now that we have some concept of *ignored* entries, I'm thinking that this change is no longer the best way to proceed.  Simply returning `None` here will be easier to filter out of the final results and thus, easier to tally the results.